### ISSUE TITLE
Unstated dependency @kadira/storybook-addon-knobs?

### ISSUE THREAD

**Issue by SereznoKot** [philcockfield/storybook-host#6]
Running storybook right after `npm i` gives me this:
```
ERROR in ./~/storybook-host/lib/index.js
Module not found: Error: Cannot resolve module '@kadira/storybook-addon-knobs'
in /home/alex/projects/preki/node_modules/storybook-host/lib
 @ ./~/storybook-host/lib/index.js 4:12-52
```
Manually `npm i @kadira/storybook-addon-knobs` fixes it. Maybe it should be moved from devDeps to regular dependencies?

**Comment by philcockfield** [comment]
Oh!  Good find @SereznoKot!!  You're right.

**Comment by philcockfield** [comment]
Published as `storybook-host@1.0.9`.  Thanks @SereznoKot
